RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -147,7 +147,9 @@
 eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
     _detachedQuadStepTrajectoryFilterBase.clone(
```

Comment:
Can I just make another branch that point to the previous commit? AFAIK, the only modifications that I want to keep after you comments are the ones in `Validation/RecoTrack/python/TrackValidation_cff.py`